release-lockfile.spec.md

```diff
@@ -276,14 +286,148 @@ this project depends on.
     * Values **must** be valid IPFS URIs which resolve to a valid *Release Lock File*
 
 
-## Object Definitions
+## Definitions
 
 Definitions for different objects used within the release lockfile.  All
 objects allow custom fields to be included.  Custom fields **should** be
 prefixed with `x-` to prevent name collisions with future versions of the
 specification.
 
 
+### The *Link Reference* Object
+
+A link reference object has the following key/value pairs.  All link references
+are assumed to be associated with some corresponding bytecode.
+
+#### Offset: `offset`
```

Comment:
(re: LinkValue below, this could be rename to `offsets` and converted to array, for generality)